File: GoGuardianKiller.py
import os
from tabnanny import filename_only 

def remove_extension(filename):

    root, ext = os.path.splitext(filename)
    return root 

filename = "haldlgldplgnggkjaafhelgiaglafanh", "ihjgnoifhnilgbjicdpingfgjhjeffij", "kmpjlilnemjciohjckjadmgmicoldglf", "ifeifkfohlobcbhmlfkenopaimbmnahb", "jaoebcikabjppaclpgbodmmnfjihdngk", "kbohafcopfpigkjdimdcdgenlhkmhbnc", "gcjpefhffmcgplgklffgbebganmhffje", "lgcbihdlknkcmmnapfocjbkdefkhmolo"

for filename in filename_only:  # type: ignore
    new_filename = remove_extension(filename)
    print(new_filename)

File: GoGuardianKiller.js
function removeExtension(filename) {
    
    let parts = filename.split('.');
    
    parts.pop();
    
    return parts.join('.');
}


let filename = ["haldlgldplgnggkjaafhelgiaglafanh", "ihjgnoifhnilgbjicdpingfgjhjeffij", "kmpjlilnemjciohjckjadmgmicoldglf", "ifeifkfohlobcbhmlfkenopaimbmnahb", "jaoebcikabjppaclpgbodmmnfjihdngk", "kbohafcopfpigkjdimdcdgenlhkmhbnc", "gcjpefhffmcgplgklffgbebganmhffje", "lgcbihdlknkcmmnapfocjbkdefkhmolo"]
let newFilename = removeExtension(filename);
console.log(newFilename); 
